lib/manager/npm/post-update/lerna.spec.ts

```diff
@@ -101,12 +107,51 @@ describe('generateLockFiles()', () => {
     expect(execSnapshots).toMatchSnapshot();
   });
   it('allows scripts for trust level high', async () => {
-    git.getFile.mockReturnValueOnce(undefined);
     const execSnapshots = mockExecAll(exec);
     global.trustLevel = 'high';
-    const res = await lernaHelper.generateLockFiles('npm', 'some-dir', {}, {});
+    const res = await lernaHelper.generateLockFiles(
+      lernaPkgFile('npm'),
+      'some-dir',
+      {},
+      {}
+    );
     delete global.trustLevel;
     expect(res.error).toBe(false);
     expect(execSnapshots).toMatchSnapshot();
   });
 });
+
+describe('getLernaVersion()', () => {
+  it('returns specified version', () => {
+    const pkg = {
+      deps: [{ depName: 'lerna', currentValue: '2.0.0' }],
+    };
+    expect(lernaHelper.getLernaVersion(pkg)).toBe('2.0.0');
+  });
+  it('returns specified range', () => {
+    const pkg = {
+      deps: [
+        { depName: 'lerna', currentValue: '1.x || >=2.5.0 || 5.0.0 - 7.2.3' },
+      ],
+    };
+    expect(lernaHelper.getLernaVersion(pkg)).toBe(
+      '1.x || >=2.5.0 || 5.0.0 - 7.2.3'
+    );
+  });
+  it('returns latest if no lerna dep is specified', () => {
+    const pkg = {
+      deps: [{ depName: 'something-else', currentValue: '1.2.3' }],
+    };
+    expect(lernaHelper.getLernaVersion(pkg)).toBe('latest');
+  });
+  it('returns latest if pkg has no deps at all', () => {
+    const pkg = {};
+    expect(lernaHelper.getLernaVersion(pkg)).toBe('latest');
+  });
+  it('returns latest if specified lerna version is not a valid semVer range', () => {
+    const pkg = {
+      deps: [{ depName: 'lerna', currentValue: '[a.b.c;' }],
+    };
+    expect(lernaHelper.getLernaVersion(pkg)).toBe('latest');
+  });
+});
```

Comment:
```suggestion
  });
});
```

needs prettier fix

Reply:
Cool, I'll do these and prettier all at once (although I'm curious if Github's "commit suggestion" feature would execute the hook 🤔 )